README.md

```diff
@@ -14,39 +14,53 @@ An overview of the architecture is shown below:
 
```

Comment:
Can we remove the image with the architecture since it's no more valid?
Not sure if it was useful even before :)